simavr/sim/avr_usb.c

```diff
@@ -684,12 +683,13 @@ avr_usb_ioctl(
 			raise_ep_interrupt(io->avr, p, ep, rxstpi);
 
 			return 0;
+		case AVR_IOCTL_USB_RESET_AND_SOFI:
+			// register SOFI every 1ms and fallthrough to reset.
+			avr_cycle_timer_register_usec(io->avr, 1000, sof_generator, p);
```

Comment:
Just to make clear it's intended. This keeps gcc happy:

```suggestion
			avr_cycle_timer_register_usec(io->avr, 1000, sof_generator, p);
			// fallthrough
```

Reply:
Thanks for this. Done.